src/sentry/notifications/utils/participants.py

```diff
@@ -384,10 +388,25 @@ def get_send_to(
     event: Event | None = None,
     notification_type: NotificationSettingTypes = NotificationSettingTypes.ISSUE_ALERTS,
     fallthrough_choice: FallthroughChoiceType | None = None,
+    rules: Iterable[Rule] | None = None,
```

Comment:
Why do we pass multiple rules here? I think we should only ever be firing one rule at a time.